crypto/src/main/scala/coop/rchain/crypto/hash/Blake2b512Block.scala

```diff
@@ -59,7 +59,9 @@ class Blake2b512Block {
                        outOffset: Int): Unit = {
     val tempChainValue: Array[Long] = new Array[Long](8)
     compress(block, inOffset, tempChainValue, true, true, false)
-    Pack.longToLittleEndian(tempChainValue, output, outOffset)
+    for (i <- 0 until 4) {
+      Pack.longToLittleEndian(tempChainValue(i), output, outOffset + i * 8)
```

Comment:
1. Current is 256 *Bits* or 32 Bytes. Previous was 512 Bits, or 64 Bytes.
2. All 8 longs (64 Bytes) are computed by compress, and we pass out the first 4.
4. Because the names or only 256 bits anyway, so we don't need the security level provided by 512 bit internal nodes.
5. Because the output array may now be smaller. We have shrunk the internal node size.